Actions/ReadSecrets/ReadSecretsHelper.psm1

```diff
@@ -131,6 +131,10 @@ function GetKeyVaultSecret {
     if ($null -eq $keyVaultCredentials) {
         return $null
     }
+    if ($secretName.Contains('_')) {
```

Comment:
Consider moving these lines after line 146.

Reply:
I guess there could be a redirection to a github secret like that - good catch